ui/pages/import-token/import-token.component.js

```diff
@@ -330,10 +366,13 @@ class ImportToken extends Component {
           type="text"
           value={customAddress}
           onChange={(e) => this.handleCustomAddressChange(e.target.value)}
-          error={customAddressError || mainnetTokenWarning}
+          error={
+            customAddressError || mainnetTokenWarning || collectibleAddressError
+          }
           fullWidth
           autoFocus
           margin="normal"
+          data-testid="custom-address-input"
         />
```

Comment:
For unit tests we can probably use getByRole for input box and avoid adding `data-testid`

Reply:
fixed here: https://github.com/MetaMask/metamask-extension/pull/13271/commits/2c635b32bc267b68a43bd0683f4be4b997fa1d58

Reply:
data-testid can also be removed

Reply:
missed this one! Thanks!